.github/workflows/post-commit.yml

```diff
@@ -0,0 +1,94 @@
+# A workflow to run against renovate-bot's PRs,
```

Comment:
nit: add a doc section to the CONTRIBUTING.md that outlines how to use this workflow.

Reply:
No existing contributing docs, but I added one.